src/Components/Search_bar/Search_bar.js

```diff
@@ -3,7 +3,7 @@ import './search_bar.css';
 
 const Searchbar = ({ handleSearchEntry, searchByWord, searchField }) => {
     return (
-        <section className="top-container">
+        <form className="top-container">
```

Comment:
I firmly feel that this form is controlled, look at that awesome function calling app's state!